docs/content/config/advanced/auth-ldap.md

```diff
@@ -6,37 +6,199 @@ title: 'Advanced | LDAP Authentication'
+    - To for example allow users from the `admin` group to spoof any sender email address, and to force everyone else to only use their personal mailbox address for outgoing email, you can use something like this: `(|(memberOf=cn=admin,*)(mail=%s))`
 
-!!! example "saslauthd"
+???+ example
 
-    - `SASLAUTHD_LDAP_FILTER`
+    A really simple `LDAP_QUERY_FILTER` configuration, using only the _user filter_ and allowing only `admin@*` to spoof any sender addresses.
 
-!!! example "dovecot"
+    ```yaml
+    - ENABLE_LDAP=1
+    - LDAP_SERVER_HOST=ldap.example.org
+    - LDAP_SEARCH_BASE=dc=example,dc=org"
+    - LDAP_BIND_DN=cn=admin,dc=example,dc=org
+    - LDAP_BIND_PW=mypassword
+    - SPOOF_PROTECTION=1
+    
+    - LDAP_QUERY_FILTER_DOMAIN=(mail=*@%s)
+    - LDAP_QUERY_FILTER_USER=(mail=%s)
+    - LDAP_QUERY_FILTER_ALIAS=(|) # doesn't match anything
+    - LDAP_QUERY_FILTER_GROUP=(|) # doesn't match anything
+    - LDAP_QUERY_FILTER_SENDERS=(|(mail=%s)(mail=admin@*))
+    ```
+
+### `DOVECOT_*_FILTER` & `DOVECOT_*_ATTRS`
+These variables specify the LDAP filters that dovecot uses to determine if a user can log in to their IMAP account, and which mailbox is responsible to receive email for a specific postfix user.
+
+This is split into the folowing two lookups, both using `%u` as the placeholder for the full login name ([see dovecot documentation for a full list of placeholders](https://doc.dovecot.org/configuration_manual/config_file/config_variables/)). Usually you only need to set `DOVECOT_USER_FILTER`, in which case it will be used for both filters.
+
+- `DOVECOT_USER_FILTER` is used to get the account details (uid, gid, home directory, quota, ...) of a user.
+- `DOVECOT_PASS_FILTER` is used to get the password information of the user, and is in pretty much all cases identical to `DOVECOT_USER_FILTER` (which is the default behaviour if left away).
+
+If your directory doesn't have the [postfix-book schema](https://github.com/variablenix/ldap-mail-schema/blob/master/postfix-book.schema) installed, then you must change the internal attribute handling for dovecot. For this you have to change the `pass_attr` and the `user_attr` mapping, as shown in the example below:
+
+```yaml
+- DOVECOT_PASS_ATTRS=<YOUR_USER_IDENTIFIER_ATTRIBUTE>=user,<YOUR_USER_PASSWORD_ATTRIBUTE>=password
+- DOVECOT_USER_ATTRS=<YOUR_USER_HOME_DIRECTORY_ATTRIBUTE>=home,<YOUR_USER_MAILSTORE_ATTRIBUTE>=mail,<YOUR_USER_MAIL_UID_ATTRIBUTE>=uid,<YOUR_USER_MAIL_GID_ATTRIBUTE>=gid
+```
+
+!!! note
+
+    For `DOVECOT_*_ATTRS`, you can replace `ldapAttr=dovecotAttr` with `=dovecotAttr=%{ldap:ldapAttr}` for more flexibility, like for example `=home=/var/mail/%{ldap:uid}` or just `=uid=5000`.
+
+    A list of dovecot attributes can be found [in the dovecot documentation](https://doc.dovecot.org/configuration_manual/authentication/user_databases_userdb/#authentication-user-database).
 
-    - `DOVECOT_USER_FILTER`
-    - `DOVECOT_PASS_FILTER`
+???+ example: "Defaults"
```

Comment:
Sorry, missed the first time. It is still broken i think we need to remove the colon
```suggestion
???+ example "Defaults"
```